src/Propel/Runtime/Adapter/Pdo/MssqlAdapter.php

```diff
@@ -300,4 +300,25 @@ public function cleanupSQL(&$sql, array &$params, Criteria $values, DatabaseMap
             }
         }
     }
+    
+    /**
+     * Returns timestamp formatter string for use in date() function.
+     *
+     * @return string
+     */
+    public function getTimestampFormatter()
+    {
+        return 'Y-m-d H:i:s:000';
```

Comment:
Can MSSSQL support `Y-m-d H:i:s.u` or is `:000` required?

Reply:
Just finished some testing and was going to create a PR, when I noticed this.

DateTime   can use `Y-m-d H:i:s.u`;
But DateTime2 needs to use `Y-m-d H:i:s`;

_(not sure how to proceeded from here)_